### DISCUSSION TITLE
只要调试时没有关闭文件树的功能，每次调试右边的窗口被拉得很小，这种情况如何解决？

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#881] [Q&A] [answered]
如下图
![image](https://github.com/ayamir/nvimdots/assets/110797126/e8dbd94f-9350-4c4f-9440-a6873117515a)

**Comment by Jint-lzxy** [comment, marked as answer]
Wait for #877 to merge, or adjust that manually:
https://github.com/ayamir/nvimdots/blob/d425bfdc4f4d1f36d0ac1b452a58ef83f055ce20/lua/core/mapping.lua#L24-L27
